srcCs/RTIDDSImpl.cs

```diff
@@ -2309,5 +2404,6 @@ public IMessagingReader CreateReader(string topic_name, IMessagingCB callback)
         private ITypeHelper<T>                  _DataTypeHelper = null;
 
         private Semaphore _pongSemaphore = null;
+        private RTIDDSLoggerDevice  _loggerDevice = new RTIDDSLoggerDevice();
```

Comment:
I would create it as null.

Reply:
Done.
It will be initialized in public bool Initialize(int argc, string[] argv)